REPORT_TEMPLATE.md

```diff
@@ -12,7 +12,18 @@ We calculated the activity of global repositories and the top 10 most active rep
 
 We calculated the activity of global developers on GitHub and the top {{sqls.developers-top.config.topN}} most active developers are as follows.
 
-{{sqls.developers-top.text}}
```

Comment:
You should not delete this line because we still need developers analysis table here, you should add the table after this one and add some description about the table.

Reply:
Okay, I have received this message, thank you for your feedback. I will make changes right away.